File: modulo1/segmento2/apunte1/Ejercicio1.java
import java.util.Scanner;

public class Ejercicio1 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

		System.out.print("Ingrese el nombre de su barrio: ");
		String barrio = keyboard.nextLine();

		System.out.print("Ingrese la dirección postal de la comisaria más cercana: ");
		String postal = keyboard.nextLine();

		System.out.print("Ingrese las entrecalles: ");
		String entrecalles = keyboard.nextLine();

		System.out.println("La comisaría de " + barrio + " se encuentra en la calle " + postal + " entre " + entrecalles + ".");
	}
}

File: modulo1/segmento2/apunte1/ejercicio1.web/ejercicio1.js
function determinarComisaria() {
	const barrio = prompt('En qué barrio vivís?', 'Igresa tu barrio');
	const direccion = prompt('Cuál es la dirección de la comisaria más cercana?', 'Igresa la dirección');
	const entre1 = prompt('Cuál es la primera entrecalle?', 'Igresa el nombre');
	const entre2 = prompt('Cuál es la segunda entrecalle?', 'Igresa el nombre');

	alert(`La comisaría de ${barrio} se encuentra en ${direccion} entre ${entre1} y ${entre2}`);
}
